packages/microcosm-www-next/src/pages/index.js

```diff
@@ -1,40 +1,152 @@
-              A user fires up the browser to take a quiz.
-            </p>
-          </figcaption>
-        </figure>
-
-        <figure className="figure -right">
-          <img src="" className="figure__graphic" alt="TODO" />
-          <figcaption className="figure__content">
-            <h3 className="figure__content__header">In Microcosm</h3>
-            <p className="figure__content__text">
+export default class IndexPage extends React.Component {
+  constructor(props) {
+    super(props)
+  }
+
+  componentDidMount() {
+    // TODO later - accessibility to alert on change
+    this.setVars()
+    this.beginObserve()
+  }
+
+  setVars() {
+    this.onMicrocosmView = true
+    this.currentSection = 1
+    this.observeOptions = {
+      root: null,
+      rootMargin: '0px 0px 100px',
+      threshold: 1.0,
+    }
+
+    this.text = document.getElementById('text')
+    this.heading = document.getElementById('heading')
+    this.subheadingTop = document.getElementById('subheading-top')
+    this.subheadingBottom = document.getElementById('subheading-bottom')
+
+    let graphics = document.querySelectorAll('[data-module="ObserveGraphic"]')
+    this.graphicsMap = this.setGraphicsMap(graphics)
+  }
+
+  setGraphicsMap(graphics) {
+    return [].slice.call(graphics).reduce((map, graphic) => {
+      map[graphic.dataset.section] = graphic
+      return map
+    }, {})
+  }
+
+  beginObserve() {
+    //create new Observer instance
+    let observer = new IntersectionObserver(
+      this.onIntersection,
+      this.observeOptions
+    )
+
+    //start observing each graphic
+    Object.values(this.graphicsMap).forEach(graphic =>
+      observer.observe(graphic)
+    )
+  }
+
+  onIntersection = observed => {
+    let entry = observed[0]
+    let section = entry.target.dataset.section
+    let notAlreadyVisible = section !== this.currentSection
+
+    if (entry.isIntersecting && notAlreadyVisible) {
+      this.currentSection = section
+      this.changeHeading()
+      this.changeText()
+      this.changeGraphic()
+    }
+  }
+
+  changeHeading() {
+    let sectionData = data[this.currentSection]
+    this.heading.innerHTML = sectionData.heading
+  }
+
+  changeText() {
+    let sectionData = data[this.currentSection]
+    this.text.innerHTML = this.onMicrocosmView
+      ? sectionData.microcosmText
+      : sectionData.browserText
+  }
+
+  changeGraphic() {
+    let section = this.currentSection
+    let graphicElement = this.graphicsMap[section]
+    let newGraphicUrl = this.onMicrocosmView
+      ? data[section].microcosmGraphicUrl
+      : data[section].browserGraphicUrl
+
+    graphicElement.setAttribute('src', newGraphicUrl)
+  }
+
+  changeButtonText(button) {
+    button.classList.toggle('-browserView', !this.onMicrocosmView)
+  }
+
+  changeSubheadingText(button) {
+    this.subheadingTop.classList.toggle('-browserView', !this.onMicrocosmView)
+    this.subheadingBottom.classList.toggle('-browserView', !this.onMicrocosmView)
```

Comment:
I am changing some text via classes (e.g. "In Microcosm", "Meanwhile, in the browser...") but not sure if I should go the JS route because of potential accessibility issues